CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.2.0] - 2020-05-20
 ### Changed
 - Add option to configure the init container image
+- Expose prometheus service endpoint if `metrics.enabled` is `true`
```

Comment:
this should go to the `[Unreleased]` section

Reply:
Oops :see_no_evil: